rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -43,11 +44,47 @@
     from rotkehlchen.api.server import APIServer
 
 
+def assert_editing_works(
+        entry: 'HistoryBaseEntry',
+        rotkehlchen_api_server: 'APIServer',
+        sequence_index: int,
+):
+    """Assert that editing any subclass of history base entry is successful done"""
+    entry.sequence_index = sequence_index
+    entry.timestamp = TimestampMS(1639924575000)
+    entry.balance = Balance(amount=FVal('1500.1'), usd_value=FVal('1499.45'))
+    entry.location_label = '0x9531C059098e3d194fF87FebB587aB07B30B1306'
+    entry.asset = A_USDT
+    entry.notes = 'Deposit stuff for staking somewhere'
```

Comment:
can you maybe change this note to make more sense

```suggestion
    entry.notes = 'Updated entry for test using type {type(entry)}'
```

or something that makes them unique